### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by jonyhy96**
typo fix
```release-note
None
```

### CODE REVIEW

File: staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func (g *genClientset) GenerateType(c *generator.Context, t *types.Type, w io.Wr
 	sw.Do(clientsetTemplate, m)
 	for _, g := range allGroups {
 		sw.Do(clientsetInterfaceImplTemplate, g)
-		// don't generated the default method if generating internalversion clientset
```
Comment (CaoShuFeng) [inline staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go]: NAK.
In k8s internalversion is a package name. It's not a typo.

Comment (jonyhy96) [reply staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go]: Thanks for your correct! :D

**Comment by jonyhy96** [comment]
wrong commit
close it

**Pull request closed without merging**